go/oasis-node/cmd/genesis/genesis.go

```diff
@@ -41,6 +47,34 @@ const (
 	cfgBlockHeight        = "height"
 	cfgChainID            = "chain.id"
 
+	// Registry config flags.
+	cfgRegistryDebugAllowUnroutableAddresses = "registry.debug.allow_unroutable_addresses"
+	cfgRegistryDebugAllowRuntimeRegistration = "registry.debug.allow_runtime_registration"
+	cfgRegistryDebugBypassStake              = "registry.debug.bypass_stake" // nolint: gosec
+
+	// Roothash config flags.
+	cfgRoundTimeout = "roothash.round_timeout"
+
```

Comment:
probably also `debug.allow_test_keys` flag?

Reply:
I talked to Yawning about this and this flag is also used in tools where the genesis document isn't available, so this will have to stay as it is :)